downloadLatest.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
I assume we can even just make this `#!/bin/sh` since the README command is using `sh`.

Reply:
Thanks, I'll switch this.